test/priority_queue.js

```diff
@@ -0,0 +1,195 @@
+var PriorityQueue       = require('../priority_queue');
+var Node                = require('../priority_queue/Node');
+var {expect, assert}    = require('chai');
+
+describe('Node', function() {
+
+    describe('# Constructor', function() {
+        it('should create an object of type Node', function() {
+            const node = new Node("value");
```

Comment:
how does a test with no asserts work?

Reply:
It's would fail if there was an error thrown. Otherwise, it's creating an object so it's good.